### PR TITLE
framework/araui: Add return check procdures.

### PR DESCRIPTION
**Pull request by tyha1115**
- In order to prevent unpredicted operations, add return value check procedures.
- When return value of the queue is NULL, it means error case (i.e, memory violations). Therefore it considered as an error case.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c3d95ec092f3f2ccb782dd61f07f6ada31d43153] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c3d95ec092f3f2ccb782dd61f07f6ada31d43153] - Code Rule Check OK.

File: framework/src/araui/core/ui_core.c [hunk 1]
```diff
@@ -218,7 +218,8 @@ static ui_error_t _ui_process_widget_recur(ui_widget_body_t *widget, uint32_t dt
 	while (!ui_widget_is_queue_empty()) {
 		curr_widget = ui_widget_queue_dequeue();
 		if (!curr_widget) {
-			continue;
+			UI_LOGE("error: curr widget is NULL!\n");
```
Comment (Taejun-Kwon) [inline framework/src/araui/core/ui_core.c]: Then this case it will return UI_OK...

File: framework/src/araui/core/ui_core.c [hunk 2]
```diff
@@ -396,15 +399,21 @@ static void _ui_update_redraw_list_recur(ui_widget_body_t *widget)
 			}
 
 			// Update previous area
-			ui_window_add_redraw_list(curr_widget->global_rect);
+			if (ui_window_add_redraw_list(curr_widget->global_rect) != UI_OK) {
```
Comment (Taejun-Kwon) [inline framework/src/araui/core/ui_core.c]: don't have to check result of translate & rotate & scale?

Comment (tyha1115) [reply framework/src/araui/core/ui_core.c]: Yes, they just do multiplication of float number.

**Comment by seinfra** [comment]
Target : [859f873dbf7e1c2390c3755343d956c4d5e6db44] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [859f873dbf7e1c2390c3755343d956c4d5e6db44] - Code Rule Check OK.

**Pull request closed without merging**